### DISCUSSION TITLE
需要提供可以查询所有成员的sql历史记录，用于审计用途

### DISCUSSION THREAD

**Discussion by zhanbinHo** [OtterMind/Chat2DB#779] [General]
![image](https://github.com/chat2db/Chat2DB/assets/20354178/ccb892d8-af2f-41cd-a4f3-933bdb509edc)

**Comment by zhanbinHo** [comment]
目前好像只能查看到自己的sql记录

**Reply by shudizhang729-cyber** [reply]
感谢你对Chat2DB的支持，
看到你内容是想了解SQL审计方面的内容，我们了解到审计方面在企业侧使用场景频次较高，企业希望掌握数据库中的数据操作的动态，因此我们发布了Chat2DB企业版内容。在满足数据库管理、开发、数据分析等需求外，增加了企业审计、权限、知识库等功能。因为使用场景的原因，目前在个人版上不具备审计此项内容。
如你对审计内容极为关注，可以与我联系18356730015 。
